cdap-docs/introduction/source/index.rst

```diff
@@ -0,0 +1,1131 @@
+     - Run Hive command using Hive CLI: ``SELECT * FROM stream_logeventsetream LIMIT 2``
+     - HiveCLI, Beeline
+
+.. list-table::
+   :widths: 15 85
+
+   * - **CDAP**
+     - ``$ execute 'select * from stream_logEventStream limit 2'``
+       ::
+
+        +===================================================================================================================+
+        | stream_ | stream_ | stream_ | stream_ | stream_ | stream_ | stream_ | stream_ | stream_ | stream_ | stream_logeve |
+        | logeven | logeven | logeven | logeven | logeven | logeven | logeven | logeven | logeven | logeven | ntstream.user |
+        | tstream | tstream | tstream | tstream | tstream | tstream | tstream | tstream | tstream | tstream | _agent: STRIN |
+        | .ts: BI | .header | .remote | .remote | .auth_u | .date:  | .reques | .status | .conten | .referr | G             |
+        | GINT    | s: map< | _host:  | _login: | ser: ST | STRING  | t: STRI | : INT   | t_lengt | er: STR |               |
+        |         | string, | STRING  |  STRING | RING    |         | NG      |         | h: INT  | ING     |               |
+        |         | string> |         |         |         |         |         |         |         |         |               |
+        +===================================================================================================================+
+        | 1428100 | {}      | 255.255 |         |         | 23/Sep/ | GET /cd | 401     | 2969    |         | Mozilla/4.0 ( |
+        | 343436  |         | .255.18 |         |         | 2014:11 | ap.html |         |         |         | compatible; M |
+        |         |         | 5       |         |         | :45:38  |  HTTP/1 |         |         |         | SIE 7.0; Wind |
+        |         |         |         |         |         | -0400   | .0      |         |         |         | ows NT 5.1)   |
+        |-------------------------------------------------------------------------------------------------------------------|
+        | 1428100 | {}      | 255.255 |         |         | 23/Sep/ | GET /cd | 401     | 2969    |         | Mozilla/4.0 ( |
+        | 483106  |         | .255.18 |         |         | 2014:11 | ap.html |         |         |         | compatible; M |
+        |         |         | 5       |         |         | :45:38  |  HTTP/1 |         |         |         | SIE 7.0; Wind |
+        |         |         |         |         |         | -0400   | .0      |         |         |         | ows NT 5.1)   |
+        +===================================================================================================================+
+
+       |non-breaking-space|
+
+.. list-table::
+   :widths: 15 65 20
+
+   * - **Current Approach**
+     - Write a code to compute the various stats: Unique, Histograms, etc.
+     - HiveServer, Beeline
+
+.. list-table::
+   :widths: 15 85
+
+   * - **CDAP**
+     - ``$ get stream-stats logEventStream limit 1000``
```

Comment:
Not sure if `$` is good to use here. Usually I see that used a prefix for commands meant to executed in bash.